db/config.cc

```diff
@@ -1202,6 +1223,13 @@ std::map<sstring, db::experimental_features_t::feature> db::experimental_feature
     };
 }
 
+std::unordered_map<sstring, locator::replication_strategy_type> db::replication_strategy_restriction_t::map() {
```

Comment:
I need a code-comment explaining where this map() is used. Not in this patch AFAICS, so calls for some small note

Reply:
This map is used to translate strings provided in scylla.yaml to enumerators. We store 2 lists of these enumerators with deprecated/obsoleted strategies and compare against these lists when a ks is created/atered.